src/core/sagas/allergiesSynopsisSagas.js

```diff
@@ -0,0 +1,25 @@
+import { takeEvery, put } from 'redux-saga/effects';
```

Comment:
same as for the reducers : your current strategy for calling external aPIs won't scale. Instead of adding a new saga for every route you want to fetch, use a single saga, and pass the url to fetch in the action payload.

Reply:
I created function which create sagas for synopsis